web-ts/src/RootSwitcher.tsx

```diff
@@ -83,6 +81,9 @@ const RootSwitcher = ({ setContentLanguage }: PropsType): ReactElement => {
       />
       <Route exact path={RoutePatterns[NOT_FOUND_ROUTE]} component={NotFoundPage} />
       <Route path={cityContentPattern} render={props => <CityContentSwitcher cities={relevantCities} {...props} />} />
+      <Redirect exact from='/' to={landingPath} />
+      <Redirect exact from={`/:cityCode`} to={cityContentPath} />
```

Comment:
This still shadows the redirect below from `/landing` to `/landing/de` I guess.

Reply:
When I testsed each ordering the outcome is the same. Maybe the more concrete route takes precedence over the general route? Anyhow, I changed the order as it was before, as it makes this more clear and added tests

Reply:
Yes, the output is the same because `/landing` is treated as city content rotue with `landing` as the city code. Then it redirects to `/landing/${language}`, just the same way as the actual landing redirect does. You can test this by e.g. using the folowing:
```
       <Redirect exact from={`/:cityCode`} to='/asdf' /> 
```
Then `/landing` is redirected to `/asdf`.